extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -472,7 +472,7 @@ private static Key readTokenDecryptionKey(OidcTenantConfig oidcConfig) {
                             (keys.get(0).getAlgorithm() == null
                                     || keys.get(0).getAlgorithm().equals(KeyEncryptionAlgorithm.RSA_OAEP.getAlgorithm()))
                             && ("enc".equals(keys.get(0).getUse()) || keys.get(0).getUse() == null)) {
-                        key = PublicJsonWebKey.class.cast(keys.get(0)).getPrivateKey();
+                        key = ((PublicJsonWebKey) keys.get(0)).getPrivateKey();
```

Comment:
How is the implicit cast better ? I don't really mind, it does not seem to be worthy an update, but let's see what other prefer

Reply:
looks like `new Function<>` is preferred, so let's keep these changes on, let me resolve, but you are all welcome to comment